client/js/components/map/admin/LayerSettings.vue

```diff
@@ -351,6 +356,64 @@ export default {
       })
     },
 
+    validateXtrasseUrl () {
+      // UX feedback via notifications only - backend does authoritative validation
```

Comment:
Here we could explain that with the current frontend architecture, it's not possible to validate the url on submit because the submit button is in a twig file